canonical-web-and-design/hiring.canonical.com
# Modify api endpoint for the 2nd chart (Performance)

## Done
Modified `api/application_review/performance` so it returns data for the performance chart
## QA
- Run server as Peter: `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- View charts on http://0.0.0.0:8501/api/application_review/performance
- Verify it returns `json`
## Issue
#72
